pwnlib/gdb.py

```diff
@@ -882,7 +883,7 @@ def find_module_addresses(binary, ssh=None, ulimit=False):
     # Get the addresses from GDB
     #
     libs = {}
-    cmd  = "gdb -q --args %s" % (binary)
+    cmd  = "%s -q --args %s" % (binary(), binary)
```

Comment:
This looks problematic, as `binary` refers to a global function and to a local variable. Please rename the local variable in order for it to work.